src/zoid/card-fields/component.jsx

```diff
@@ -66,10 +66,11 @@ export type CardFieldComponent = ZoidComponent<CardFieldProps>;
 
 type CardFieldsExports = {|
     submit : () => ZalgoPromise<void>,
-    setAttribute : () => void,
-    removeAttribute : () => void,
-    addClass : () => void,
-    removeClass : () => void
+    setAttribute : () => ZalgoPromise<void>,
+    removeAttribute : () => ZalgoPromise<void>,
+    addClass : () => ZalgoPromise<void>,
+    removeClass : () => ZalgoPromise<void>,
+    getState : () => ZalgoPromise<void>
```

Comment:
Should this one return an object?
```suggestion
    getState : () => ZalgoPromise<Object>
```

Reply:
Yes you are right, it should. Good catch!